packages/@aws-cdk-testing/framework-integ/test/aws-route53/test/integ.route53-imported-delegation.ts

```diff
@@ -0,0 +1,21 @@
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as cdk from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { PublicHostedZone } from 'aws-cdk-lib/aws-route53';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-route53-imported-delegation-integ');
+
+const role = new iam.Role(stack, 'Role', {
+  assumedBy: new iam.AccountRootPrincipal(),
+});
+
+const publicZone = PublicHostedZone.fromPublicHostedZoneId(stack, 'PublicZone', 'public-zone-id');
+publicZone.grantDelegation(role);
```

Comment:
I think we can move these lines of code into the [cross-account-zone-delegation integ test](https://github.com/aws/aws-cdk/blob/0bcc4b4b9d0a3dab29be343c4c3db4da7bbde00a/packages/%40aws-cdk-testing/framework-integ/test/aws-route53/test/integ.cross-account-zone-delegation.ts), unless you thought about this and had a reason for keeping it separate.

Maybe the name could to shift to `const importedPublicZone = PublicHostedZone.from...`